src/sql/workbench/browser/modelComponents/editor.component.ts

```diff
@@ -66,7 +68,7 @@ export default class EditorComponent extends ComponentBase implements IComponent
 		this._editor.create(this._el.nativeElement);
 		this._editor.setVisible(true);
 		let uri = this.createUri();
-		this._editorInput = this._instantiationService.createInstance(UntitledTextEditorInput, uri, false, 'plaintext', '', '');
+		this._editorInput = this.editorService.createInput({ forceUntitled: true, resource: uri, mode: 'plaintext' }) as UntitledTextEditorInput;
```

Comment:
vscode changes how untitled inputs are created. Now the inputs expect their model as a constructor input. Inputs are what are created from the untitledtexteditorservice, not inputs. The editor service can create input and models for us. I'm not sure on the side affects of this. @chlafreniere another reason to use a custom input/model for notebook cells.